pkgs/applications/networking/remote/xrdp/dynamic_config.patch

```diff
@@ -179,8 +179,21 @@ index 8fa34aea..da94cf95 100644
                  list_delete(items);
                  list_delete(values);
                  return 1;
+diff --git a/sesman/chansrv/chansrv.c b/sesman/chansrv/chansrv.c
+index ef3960ed..7d167728 100644
+--- a/sesman/chansrv/chansrv.c
++++ b/sesman/chansrv/chansrv.c
+@@ -1706,7 +1706,7 @@ main(int argc, char **argv)
+     /*
+      * The user is unable at present to override the sysadmin-provided
+      * sesman.ini location */
+-    config_path = XRDP_CFG_PATH "/sesman.ini";
++    config_path = argv[2];
```

Comment:
Isn't it better to pass the value as an environment variable?

That way we can easily override it on units without a probable risk of having to workaround stuff to maintain CLI compatibility. It's simpler.

Reply:
If the arguments change, we will have merge conflicts with this patch anyway. The rest of the patch is written around passing an extra argument with the config location, I don't think it makes sense do things differently here.